pkg/reporting/cookbooks_test.go

```diff
@@ -257,6 +257,9 @@ func TestCookbooks(t *testing.T) {
 				default:
 					t.Fatal("unexpected cookbook name")
 				}
+				// TODO @afiune uncomment when CookbookMock DownloadTo() creates a local dir
```

Comment:
Is there a way to test this that could avoid having the the mock touch the local filesystem?

Reply:
I think that it is safe to create the directory once all our tests are running on temp directories, we should fix that with https://github.com/chef/chef-analyze/issues/57